src/main/java/org/jboss/ejb/client/NamingEJBClientInterceptor.java

```diff
@@ -183,6 +199,9 @@ static boolean setNamingDestination(final AbstractInvocationContext context, fin
                 result.add(check);
             }
         }
+        if (Logs.INVOCATION.isDebugEnabled()) {
+            Logs.INVOCATION.debugf("NamingEJBClientInterceptor: computing preffered URIs: URIs = %s, preferred URIs = %s", uris, result);
```

Comment:
Another typo: preffered instead of preferred

Reply:
Done